state.go

```diff
@@ -835,6 +834,18 @@ func (cw *CheckWaitState) Handle(ctx *StateContext, c Controller) (State, bool)
 		return cw.Wait(next.state, cw, wait)
```

Comment:
We need to update `lastInventoryUpdateAttempt` and `lastUpdateCheckAttempt` also inside this if clause.

Otherwise such variables will never be updated in the normal workflow (i.e. when we want to actually wait the interval).